ui-src/index.js

```diff
@@ -53,15 +53,41 @@ function req(url) {
   });
 }
 
-// Produces a human-readable but unambiguous format of the given timestamp:
-// ISO-8601 plus an extra colon component after the seconds indicating the
-// fractional time in 90,000ths of a second.
-function formatTime(ts90k) {
-  const m = moment.tz(ts90k / 90, zone);
-  const frac = ts90k % 90000;
-  return m.format('YYYY-MM-DDTHH:mm:ss:' + String(100000 + frac).substr(1) + 'Z');
+/**
+ * Produces a human-readable timestamp in 90k units.
+ *
+ * The format is ISO-8601 plus an extra colon component after the seconds
```

Comment:
"The default format", maybe, as it's customizable now? And the next couple paragraphs seem to reflect the older version that did microseconds (also there's a typo: "preision").